script.js

```diff
@@ -0,0 +1,18 @@
+function removeTransition(e) {
+  console.log(e.propertyName);
```

Comment:
There is no need to log this property in the result version. Remember to remove dev-issues before pushing :)

Reply:
DONE!